imports/plugins/custom/gondor-theme/client/styles/main.less

```diff
@@ -0,0 +1,27 @@
+.admin-shortcut-bar nav .btn.active {
+  outline-color: @rui-brand-dark-disabled;
+}
+
+.btn-default.active {
+  outline-color: @rui-brand-dark-disabled;
+}
+
+.btn-default:focus, .btn-default.focus,
+.btn-default:hover, .btn-default.hover,
+.btn-default:active, .btn-default.active {
+  color: #432c4a;
+  background-color: @rui-default-bg;
+  border-color: #432c4a;
```

Comment:
Good work. I noticed you have been using variables to reference color codes. It will be nice if you maintain that culture throughout the codebase and have all color codes referenced with variables.